src/kademlia/routing_table.cpp

```diff
@@ -127,7 +127,7 @@ int routing_table::bucket_limit(int bucket) const
 {
 	if (!m_settings.extended_routing_table) return m_bucket_size;
 
-	static const aux::array<int, 4> size_exceptions({{16, 8, 4, 2}});
+	static const aux::array<int, 4> size_exceptions{{{16, 8, 4, 2}}};
```

Comment:
yes, I tried and it doesn't work, `std::array` has no explicit constructor and I was unable to achieve the same using `std:: initializer_list`